components/suggestion/modal_suggestion_list.tsx

```diff
@@ -39,7 +80,7 @@ export default class ModalSuggestionList extends React.PureComponent {
         return {top: 0, bottom: 0, width: 0};
     }
 
-    onModalScroll = (e) => {
+    onModalScroll = (e: any) => {
```

Comment:
This should be something like `ScrollEvent`

Reply:
yes,
it can be `ChangeEvent<HTMLDivElement>`

Reply:
This needs to be a generic `Event` because it's added after the fact using `addEventListener` instead of being added via a React prop